src/bin/casr-python.rs

```diff
@@ -104,6 +105,13 @@ fn main() -> Result<()> {
                 .value_name("FILE")
                 .help("Stdin file for program"),
         )
+        .arg(
+            Arg::new("ignore")
```

Comment:
Also, pass ignore parameter in `call_casr_san` function when calling casr-san.

Reply:
done